editor/src/utils/feature-switches.ts

```diff
@@ -55,7 +57,8 @@ let FeatureSwitches: { [feature in FeatureName]: boolean } = {
   'Canvas Selective Rerender': true,
   'Single child, contiguous parent: move parent': true,
   'Single child, zero sized parent: move parent': true,
-  'Resize image on drop': true,
+  'Resize image on drop': false,
+  'Show preview on drop': false,
```

Comment:
was there a decision about these two feature switches? if there was, please remove them and the related (now disabled) code changes!

Reply:
done!